src/intersectionWith.js

```diff
@@ -12,7 +12,7 @@ var uniqWith = require('./uniqWith');
  * @memberOf R
  * @since v0.1.0
  * @category Relation
- * @sig (a -> a -> Boolean) -> [*] -> [*] -> [*]
+ * @sig ((a, a) -> Boolean) -> [a] -> [a] -> [a]
```

Comment:
I suppose `((a, b) -> Boolean) -> [a] -> [b] -> [a]` is more in line with what the code actually does, correct? `((a, a) -> Boolean) -> [a] -> [a] -> [a]` is more inline with "ideal usage."

Maybe the appropriate question to ask is what's more useful for API consumers?